src/main/java/com/github/jscookie/javacookie/Cookies.java

```diff
@@ -65,6 +65,9 @@ public synchronized String get( String name ) {
 	@Override
 	public <T> T get( String name, Class<T> dataType ) throws CookieParseException {
 		String value = get( name );
+		if(value == null) {
```

Comment:
This should be `if ( value == null )` to be consistent with the rest of the file
```suggestion
		if ( value == null ) {
```

It might be worth only release a breaking v1 once we fix this NullPointer hell. Perhaps introducing a [NullObjectPattern](https://en.wikipedia.org/wiki/Null_object_pattern)?